app/views/pages/report.ejs

```diff
@@ -36,396 +36,445 @@
+      <div class="hashtag-item flex flex-col items-center w-1/2">
+        <div class="hashtag-item__title text-cvs-red font-medium text-xl md:text-3xl uppercase m-2">#washyourhands</div>
+        <img src="/static/images/washing-hands.svg" alt=""
+          class="hashtag-item__img h-48 w-48 lg:h-64 lg:w-64 m-2">
       </div>
-    </section>
+      <div class="hashtag-item flex flex-col items-center w-1/2">
+        <div class="hashtag-item__title text-cvs-red font-medium text-xl md:text-3xl uppercase m-2">#staythefuckhome</div>
+        <img src="/static/images/stay-home.svg" alt=""
+          class="hashtag-item__img h-48 w-48 lg:h-64 lg:w-64 m-2">
+      </div>
+    </div>
   </div>
-  <%- include('partials/footer') -%>
-  <script src="/static/form.js?v=<%= locals.cacheKey %>"></script>
-  <script>
-    function hasSubmittedAlertOnClick() {
-        document.getElementById('has-submitted-alert').style.display = 'none';
-        window.localStorage.setItem("hasCrossedOutInfo", "true");
-    }
-
-    if ((window.localStorage.getItem("hasSubmittedProfile") === "true"
-        || window.localStorage.getItem("isUsingPasscodeLink") === "true")
-        && window.localStorage.getItem("hasCrossedOutInfo") !== "true"
-        && window.location.pathname.indexOf("helsetilstand") === -1) {
-        document.getElementById('has-submitted-alert').style.display = 'flex';
-    }
-
-    var symptoms = document.querySelectorAll('input[name^="symptom-"]:checked');
-    if (symptoms != null && symptoms.length > 0) {
-      document.getElementById('symptom-start-question').style.display = 'block';
-    }
-
-    changeTestResultDisplay(document.getElementById("been-tested-yes").checked);
-    // Remove query params from url
-    if (window.history.replaceState) {
-      window.history.replaceState({}, document.title, window.location.pathname);
-    }
-  </script>
-  <script src="/static/hamburger-menu.js?v=<%= locals.cacheKey %>"></script>
-</body>
-</html>
+</section>
+
+<script src="/static/form.js?v=<%= locals.cacheKey %>"></script>
+
+<script>
+  function hasSubmittedAlertOnClick() {
+      document.getElementById('has-submitted-alert').style.display = 'none';
+      window.localStorage.setItem("hasCrossedOutInfo", "true");
+  }
+
+  if ((window.localStorage.getItem("hasSubmittedProfile") === "true"
+      || window.localStorage.getItem("isUsingPasscodeLink") === "true")
+      && window.localStorage.getItem("hasCrossedOutInfo") !== "true"
+      && window.location.pathname.indexOf("helsetilstand") === -1) {
+      document.getElementById('has-submitted-alert').style.display = 'flex';
+  }
+
+  var symptoms = document.querySelectorAll('input[name^="symptom-"]:checked');
+  if (symptoms != null && symptoms.length > 0) {
+    document.getElementById('symptom-start-question').style.display = 'block';
+  }
+
+  changeTestResultDisplay(document.getElementById("been-tested-yes").checked);
```

Comment:
maybe update this function to make #test-result-question required

Reply:
would you kind writing something? happy to copy paste of give you access to this branch

Reply:
sure, If you'll give me access ill push some code

Reply:
@ian-stats: here you go: https://github.com/BustByte/coronastatus/invitations